metadata/decoration.xml

```diff
@@ -1,37 +1,32 @@
 <?xml version="1.0"?>
 <wayfire>
 	<plugin name="decoration">
-	<_short>Decoration</_short>
-	<category>Effects</category>
-	<option name="font" type="string">
-		<_short>Font</_short>
-		<default>serif</default>
-	</option>
-
-  <option name="title_height" type="int">
-    <_short>Title height</_short>
-    <default>30</default>
-  </option>
-
-  <option name="border_size" type="int">
-    <_short>Border size</_short>
-    <default>4</default>
-  </option>
-
-  <option name="active_color" type="color">
-    <_short>Color when window is active</_short>
-    <default>#222222AA</default>
-  </option>
-
-  <option name="inactive_color" type="color">
-    <_short>Color when window is inactive</_short>
-    <default>#333333DD</default>
-  </option>
-
-  <option name="button_order" type="string">
-    <_short>Order of window buttons</_short>
-    <default>minimize maximize close</default>
-  </option>
-
-  </plugin>
+		<_short>Decoration</_short>
+		<category>Effects</category>
+		<option name="font" type="string">
+			<_short>Font</_short>
+			<default>sans-serif</default>
+		</option>
+		<option name="title_height" type="int">
+			<_short>Title bar height</_short>
+			<default>30</default>
+		</option>
+		<option name="border_size" type="int">
+			<_short>Border size</_short>
+			<default>4</default>
+		</option>
+		<option name="button_order" type="string">
+			<_short>Order of window buttons</_short>
+			<default>minimize maximize close</default>
+		</option>
+		<!-- Colors -->
+		<option name="active_color" type="color">
+			<_short>Color when window is active</_short>
+			<default>0.0 0.0 0.0 0.6</default>
```

Comment:
Sorry, I forgot to mention this yesterday. The default colors seem extremely black and rely on transparency only to differentiate between active and inactive. I cannot notice the difference with a dark blackground (I guess you tested this on a white background, hence it worked well for you).

Also, you heavily use alpha, which I don't like (personal opinion :P)

So, can you choose something different for inactive color (can also be dark, just not pure black again), and use a little bit higher alpha in both colors?